consensus/src/sync/syncer.rs

```diff
@@ -185,12 +197,17 @@ impl<N: Network, M: MacroSync<N::PeerId>, L: LiveSync<N>> Syncer<N, M, L> {
     pub fn move_peer_into_live_sync(&mut self, peer_id: N::PeerId) {
         debug!(%peer_id, "Adding peer to live sync");
         self.live_sync.add_peer(peer_id);
+        self.events.send(SyncEvent::AddLiveSync(peer_id)).ok();
```

Comment:
```suggestion
        let _ = self.events.send(SyncEvent::AddLiveSync(peer_id));
```

Using `.ok()` doesn't communicate that you want to ignore the error. `.ok()` is used to convert the `Ok` case of a `Result<T, E>` into an `Option<T>`.